nifi-nar-bundles/nifi-cassandra-bundle/nifi-cassandra-services-nar/src/main/resources/META-INF/NOTICE

```diff
@@ -0,0 +1,291 @@
+  	Java SE, which can be obtained at:
+
+  	  * LICENSE:
+  	    * license/LICENSE.harmony.txt (Apache License 2.0)
+  	  * HOMEPAGE:
+  	    * http://archive.apache.org/dist/harmony/
+
+  	This product contains a modified portion of 'jbzip2', a Java bzip2 compression
+  	and decompression library written by Matthew J. Francis. It can be obtained at:
+
+  	  * LICENSE:
+  	    * license/LICENSE.jbzip2.txt (MIT License)
+  	  * HOMEPAGE:
+  	    * https://code.google.com/p/jbzip2/
+
+  	This product contains a modified portion of 'libdivsufsort', a C API library to construct
+  	the suffix array and the Burrows-Wheeler transformed string for any input string of
+  	a constant-size alphabet written by Yuta Mori. It can be obtained at:
+
+  	  * LICENSE:
+  	    * license/LICENSE.libdivsufsort.txt (MIT License)
+  	  * HOMEPAGE:
+  	    * https://github.com/y-256/libdivsufsort
+
+  	This product contains a modified portion of Nitsan Wakart's 'JCTools', Java Concurrency Tools for the JVM,
+  	 which can be obtained at:
+
+  	  * LICENSE:
+  	    * license/LICENSE.jctools.txt (ASL2 License)
+  	  * HOMEPAGE:
+  	    * https://github.com/JCTools/JCTools
+
+  	This product optionally depends on 'JZlib', a re-implementation of zlib in
+  	pure Java, which can be obtained at:
+
+  	  * LICENSE:
+  	    * license/LICENSE.jzlib.txt (BSD style License)
+  	  * HOMEPAGE:
+  	    * http://www.jcraft.com/jzlib/
+
+  	This product optionally depends on 'Compress-LZF', a Java library for encoding and
+  	decoding data in LZF format, written by Tatu Saloranta. It can be obtained at:
+
+  	  * LICENSE:
+  	    * license/LICENSE.compress-lzf.txt (Apache License 2.0)
+  	  * HOMEPAGE:
+  	    * https://github.com/ning/compress
+
+  	This product optionally depends on 'lz4', a LZ4 Java compression
+  	and decompression library written by Adrien Grand. It can be obtained at:
+
+  	  * LICENSE:
+  	    * license/LICENSE.lz4.txt (Apache License 2.0)
+  	  * HOMEPAGE:
+  	    * https://github.com/jpountz/lz4-java
+
+  	This product optionally depends on 'lzma-java', a LZMA Java compression
+  	and decompression library, which can be obtained at:
+
+  	  * LICENSE:
+  	    * license/LICENSE.lzma-java.txt (Apache License 2.0)
+  	  * HOMEPAGE:
+  	    * https://github.com/jponge/lzma-java
+
+  	This product contains a modified portion of 'jfastlz', a Java port of FastLZ compression
+  	and decompression library written by William Kinney. It can be obtained at:
+
+  	  * LICENSE:
+  	    * license/LICENSE.jfastlz.txt (MIT License)
+  	  * HOMEPAGE:
+  	    * https://code.google.com/p/jfastlz/
+
+  	This product contains a modified portion of and optionally depends on 'Protocol Buffers', Google's data
+  	interchange format, which can be obtained at:
+
+  	  * LICENSE:
+  	    * license/LICENSE.protobuf.txt (New BSD License)
+  	  * HOMEPAGE:
+  	    * https://github.com/google/protobuf
+
+  	This product optionally depends on 'Bouncy Castle Crypto APIs' to generate

```

Comment:
The bouncy castle reference for `nifi-cassandra-services-nar` is here.

Reply:
I might have read that as a different indentation level. Either way, as IANAL, works for me since it does list that.